src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,92 @@
+// ----- BASE ELEMENTS ---------------
+
+.navbar {
```

Comment:
Please change to `.eyebrow {`.  This class name may change, so we'll update if needed.

Reply:
Changes have been done, Please review